website/content/features.ts

```diff
@@ -20,18 +20,6 @@ export type Feature = {
 export type Features = Array<Feature>;
 
 const FEATURES: Features = [
-  {
-    name: "Cloud asset inventory",
-    description: `Build your own multi-cloud asset inventory with standard SQL and BI tools.`,
-    Icon: ChartBarIcon,
-    page: "all",
-  },
-  {
-    name: "CSPM",
-    description: `Customize pre-built open source SQL policies and visualize them with your any of your favorite BI tools.`,
-    Icon: KeyIcon,
-    page: "all",
-  },
   {
     name: "Open source",
     description: `Easily extensible plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.`,
```

Comment:
```suggestion
    description: `Extensible plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.`,
```

I know this wasn't changed in the PR, but upon seeing it I realized that "easily extensible" doesn't seem right.